tests-invoke

```diff
@@ -68,12 +69,13 @@ def main():
 
 
 class PullTask(object):
-    def __init__(self, name, revision, context, test_project, test_branch):
+    def __init__(self, name, revision, context, test_project, test_branch, github_context):
```

Comment:
What's the difference between context and github_context now? Is the github_context the full string?

Reply:
`os/scenario[@Bots#<pr_number>][@org/project[/branch]]`
context is `os/scenario`, github_context is `os/scenario[@Bots#<pr_number>][@org/project[/branch]]`